readme.md

```diff
@@ -16,10 +16,15 @@ source ~/bin/migbin/bashrc
 ---
```

Comment:
Great stuff, Don't forget to update here if you change function names or combine secureit with fixlinks

Reply:
updated